<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MountMB</class>
 <widget class="QMainWindow" name="MountMB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>570</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FPA/MB Mount Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelHeader">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>12</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>MS-177 FPA/MB Mount Calculator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelImg1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>550</width>
      <height>235</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/images/sbf40010/SHARED/DEWAR/PUBLIC/DF/177/calcSupport/mb3.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>550</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Control Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>155</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SCA 1 Top Fiducial</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFooter2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>540</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFooter3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>530</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>s/b 11.230° ± .300</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputAngle">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>530</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditControl">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>XXXXXXXXXX</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSCA1y">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>X.XXXX</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSCA1z">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>X.XXXX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSave">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save Data</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>415</x>
      <y>90</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear Data</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCalc">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>460</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>550</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>585</y>
      <width>550</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSCA2z">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>475</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>X.XXXX</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSCA2y">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>445</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>X.XXXX</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputCenter">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>615</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelFooter6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>615</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>s/b 0.0130 ± .002</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelFooter5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>625</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFooter1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>FPA Angle</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFooter4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>615</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>-A- to Optical Center Height</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>475</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>445</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SCA 2 Bottom Fiducial</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoad">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Load Data</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Dewar Serial Number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSerial">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>XXX</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditControl</tabstop>
  <tabstop>lineEditSerial</tabstop>
  <tabstop>pushButtonLoad</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>pushButtonClear</tabstop>
  <tabstop>lineEditSCA1y</tabstop>
  <tabstop>lineEditSCA1z</tabstop>
  <tabstop>lineEditSCA2y</tabstop>
  <tabstop>lineEditSCA2z</tabstop>
  <tabstop>pushButtonCalc</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonSave</sender>
   <signal>clicked()</signal>
   <receiver>MountMB</receiver>
   <slot>saveData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>MountMB</receiver>
   <slot>clearData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCalc</sender>
   <signal>clicked()</signal>
   <receiver>MountMB</receiver>
   <slot>calculateData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonLoad</sender>
   <signal>clicked()</signal>
   <receiver>MountMB</receiver>
   <slot>loadData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadData()</slot>
  <slot>saveData()</slot>
  <slot>clearData()</slot>
  <slot>calculateData()</slot>
 </slots>
</ui>
